src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+  background-color: gray;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  /* width: 100%; */
+  padding: 0 50px;
+}
+
+.logo__img {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  position: relative;
+}
+
+.nav__link {
+  display: block;
+  color: #000;
+  text-decoration: none;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. just use padding instead